### ISSUE TITLE
Date Picker: Accessibility for Month & Year

### ISSUE THREAD

**Issue by sparkypro** [SAP/ui5-webcomponents#5421]
**Describe the bug**
The ‘Next’ and ‘Previous’ buttons within the calendar are not reachable via keyboard

**UI Link**
visit https://sap.github.io/ui5-webcomponents-react/?path=/docs/inputs-datepicker--default-story

**To Reproduce**
1. Go to DatePicker
2. Use keyboard tab key to reach month
3. The table of months is not read out with screen reader
4. The table of year is not read out with screen reader

**Expected behavior**
The ‘Next’ and ‘Previous’ buttons within the calendar are not reachable via keyboard

**Screenshots**
<img width="103" alt="image" src="https://user-images.githubusercontent.com/108118034/176108592-d2335e61-a1a5-4b7c-97fd-2b6886e9134b.png">

**UI5 Web Components for React Information**
`@ui5/webcomponents` version:1.3.1
`@ui5/webcomponents-react` version: 0.23.2
Operating System: Windows 11
Browser: Chrome
